gatsby/plugins/gatsby-source-bnf/src/field-extensions/slug.test.ts

```diff
@@ -31,6 +31,34 @@ describe("slug field extension", () => {
 				"nurse-prescribers-formulary-analgesics"
 			);
 		});
+
+		it("should strip HTML and brackets", () => {
+			expect(
+				slugify("<span>Anti-D (Rh<sub>0</sub>) immunoglobulin</span>")
+			).toBe("anti-d-rh0-immunoglobulin");
+		});
+
+		it("should retain letter number combos", () => {
+			expect(slugify("C1-esterase inhibitor")).toBe("c1-esterase-inhibitor");
+		});
+
+		it("should replace apostrophes with hyphens", () => {
```

Comment:
description doesn't reflect the assertion.... I would expect parkinson-s-disease based on that desc

Reply:
Good spot, will sort!

Reply:
Sorted!